source/tutorials/thermal-daemon/thermal-daemon.rst

```diff
@@ -86,3 +86,27 @@ Cooling device   soc power limit 1
 +----------+-------------------+-----------------------------------------+
 | Critical | t >= 104          | Shutdown the system                     |
 +----------+-------------------+-----------------------------------------+
+
+Whiskey Lake
+~~~~~~~~~~~~~~
+
+- Platform Thermal Parameters
+
+================ =========================
+Tjmax 	         100 (degree celsisus)
+Thermal Zone     x86_pkg_temp
+Cooling device   soc power limit 1
+================ =========================
+
+- Thermal Management Policy
```

Comment:
The bullet list must be ended with an empty line. You have to add a blank line below.

Reply:
Thank you @ttzeng! Just updated, please help to review it.